modules/installing-ocp-agent.adoc

```diff
@@ -178,6 +178,11 @@ You must provide the rendezvous IP address when you do not specify at least one
 $ openshift-install --dir <install_directory> agent create image
 ----
 +
+[IMPORTANT]
+====
+The `dir` parameter specifies the assets directory, which stores the manifest files, the ISO image, and ** auth ** directory. The ** auth ** directory stores the ** kubeadmin-password ** and ** kubeconfig ** files. As a `kubeadmin` user, you can use `kubeconfig` to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. Note that the `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. You can use the contents of ** kubeadmin-password ** file to log into the console during the bootstrap as the `core` user to perform debugging. 
```

Comment:
One thing I found a bit confusing about the latest change is that its not obvious the the `auth` directory is created when the user runs the agent create image command. I recommend stating that, so I user doesn't interpret that they should put a password here.
Also in the setting `export KUBECONFIG=<install_directory>/auth/kubeconfig`its not clear that `install_directory `is the same as the `dir` parameter.

Reply:
Listing the uses of the kubeadmin-password file feels a bit developer-centric. From the user's perspective, they'll need this info when they need to log in to debug prior to or during bootstrapping.

So it would be better to have a section on how to debug, where the answer is (1) use ssh to log in as the `core` user using the private key corresponding to the public key you provided in the install-config, or (2) failing that (e.g. because the network is not working) you can log into the hardware console as the `core` user using the password saved in the kubeadmin-password file in the `auth` subdirectory of the install directory. It might pay to also mention that some status information is [displayed on the login screen](https://issues.redhat.com/browse/OCPBUGS-3032) of the hardware console.

Similarly, we could have a section about how to access the cluster after installation. This is actually described in the sample installer output on ll. 222-236, so after that might be a good place. I think since that output was captured we also added the actual password to it:

```
INFO Cluster is installed
INFO Install complete!
INFO To access the cluster as the system:admin user when using 'oc', run
INFO     export KUBECONFIG=<installer dir>/auth/kubeconfig
INFO Access the OpenShift web-console here: https://console-openshift-console.apps.sno-cluster.test.example.com
INFO Login to the console with user: "kubeadmin", and password: <provided>
```

Minor technical point: the username to use (with kubeadmin-password) to [log in to the OpenShift Console](https://docs.openshift.com/container-platform/4.12/web_console/web-console.html#web-console-overview_web-console) is `kubeadmin`; the user for which the `kubeconfig` file contains the credentials is called just `admin`.